.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,53 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: '[Bug Report]'
+labels: bug
+assignees: ''
+
+---
+
+------------------------------- delete below -------------------------------
+
+INSTRUCTIONS
+==========
+
+Please follow the instructions and template below to save us time requesting additional information from you. For more information on all of your support options, please see [Azure IoT support and help options](https://docs.microsoft.com/en-us/azure/iot-fundamentals/iot-support-help).
+
+*If this is an issue encountered by following a Microsoft document, please open an issue on that page itself. This ensures the fastest targeted response, and you can do this by scrolling all the way down to the bottom of the page to the **Feedback** section to report the issue.*
+
+1. Search existing issues to avoid creating duplicates.
+
+2. If possible test using the [latest release](https://github.com/Azure/azure-iot-sdk-csharp/releases) to make sure your issues has not already been fixed.
+
+3. Do not share information from your Azure subscription here (connection strings, service names (IoT Hub name, Device Provisioning Service scope ID), etc...). If you need to share any of this information, you can create a ticket and [get assistance from Microsoft Support](https://docs.microsoft.com/en-us/azure/azure-supportability/how-to-create-azure-support-request).
+
+4. Include enough information for us to address the bug:
+   -  A detailed description.
+   -  A [Minimal Complete Reproducible Example](https://stackoverflow.com/help/mcve). This is code we can cut and paste into a readily available sample and run, or a link to a project you've written that we can compile to reproduce the bug. 
+   -  Console logs.
```

Comment:
Add a link to this readme: https://github.com/Azure/azure-iot-sdk-csharp/tree/master/tools/CaptureLogs

Reply:
added, thanks